src/protocol.rs

```diff
@@ -10,23 +10,23 @@ use arrayvec::ArrayVec;
 use bitvec::prelude::*;
 pub use byte_angle::ByteAngle;
 use byteorder::{BigEndian, ReadBytesExt, WriteBytesExt};
+pub use node::Node;
 use serde::de::DeserializeOwned;
 use serde::Serialize;
 use uuid::Uuid;
 pub use var_int::VarInt;
 pub use var_long::VarLong;
-pub use node::Node;
 use vek::{Vec2, Vec3, Vec4};
 
 use crate::entity::EntityId;
 use crate::nbt;
 
 mod byte_angle;
 pub mod codec;
+pub mod node;
 pub mod packets;
 mod var_int;
 mod var_long;
-pub mod node;
```

Comment:
It seems, with macros, order of modules is important. Try putting `node` under `packets`.
```rust
mod byte_angle;
pub mod codec;
pub mod packets;
mod var_int;
mod var_long;

pub mod node;
```

Reply:
that helped, thanks!